sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -306,8 +306,8 @@ def test_upretty_modifiers():
     assert upretty( Symbol('Fhat') ) == u('F̂')
     assert upretty( Symbol('Fbar') ) == u('F̅')
     assert upretty( Symbol('Fvec') ) == u('F⃗')
-    assert upretty( Symbol('Fprime') ) == u('F ̍')
-    assert upretty( Symbol('Fprm') ) == u('F ̍')
+    assert upretty( Symbol('Fprime') ) == u('F′')
+    assert upretty( Symbol('Fprm') ) == u('F′')
```

Comment:
The prime should be written in a way that distinguishes it from the single quote. E.g. in my browser, this looks like `'F''`, i.e. a quote, F, and two quotes - which looks like invalid Python syntax.